src/lib/index.js

```diff
@@ -12,6 +12,7 @@
 var d3 = require('d3');
 
 var lib = module.exports = {};
+var config = require('../plot_api/plot_config');
```

Comment:
Yep. That's the cleanest possible way at the moment. Mutating an internal object isn't great. But that will do.